docs/self-managed/setup/overview.md

```diff
@@ -31,8 +31,10 @@ For details on supported environments (e.g. Java or Elasticsearch versions), see
 
 You have the following options to run or deploy the above components in a self-managed fashion:
 
-- **Helm/Kubernetes**: We strongly recommend using Kubernetes to run Camunda 8 in production. In addition to stock Kubernetes we also officially support variety of providers like Red Hat OpenShift and Amazon EKS. Also using Kubernetes with Minikube or KIND can be an interesting environment to run Camunda 8 locally on developer machines.
+- **Kubernetes**: We strongly recommend using Kubernetes to run Camunda 8 in production. We provide supported Helm charts to deploy Camunda 8 into your [CNCF Certified Kubernetes](https://www.cncf.io/training/certification/software-conformance/) clusters, including a variety of providers like RedHat, OpenShift, and Amazon EKS. Consider using [minikube](https://minikube.sigs.k8s.io/docs/start/?arch=%2Fmacos%2Fx86-64%2Fstable%2Fbinary+download) or [kind](https://kind.sigs.k8s.io/docs/user/quick-start) to run Camunda 8 locally on developer machines.
```

Comment:
"We provide supported Helm charts to deploy Camunda 8 into your [CNCF Certified Kubernetes](https://www.cncf.io/training/certification/software-conformance/) clusters, including a variety of providers like RedHat, OpenShift, and Amazon EKS."  I would probably not list the vendors here, as you can find them also in the linked resource of CNCF. Also, besides OpenShift, we also don't have any vendor specifics in the helm chart. For AKS,EKS and GCP we provide guides how to configure the helm chart correctly via the value file.

Reply:
We list the most popular distributions that we test to give customers a bit more understanding of what do we mean.  This is why we provide examples: "like Red Hat OpenShift and Amazon EKS"  and not a comprehensive list.